jenkins/jenkins-install.sh

```diff
@@ -34,6 +34,10 @@ else
     echo "Jenkins installed."  
 fi
 
+echo "Determining if port 8080 is in use..."
+# Uses Netstat to list active internet connections.  This is piped to grep that 
+# looks for the LISTEN state of a port.  When an entry is found it pipes the entry
+# to another grep to examine the entry for port 8080.
 if ( netstat -an | grep -w LISTEN | grep -w :8080 ) >/dev/null
```

Comment:
This wont act how you think it should. it relies on a tcp6 connection.
`man grep | cat | grep "\-\-word\-regexp" -A 2`